server/api/profiles.js

```diff
@@ -38,16 +38,24 @@ router.post(
   }
 );
 
-router.get("/", (req, res) => {
-  getAllProfiles()
-    .then((data) => {
-      res.send(data);
-    })
-    .catch((err) => {
-      console.error(err);
-      res.send(500);
-    });
-});
+router.get(
+  "/",
+  passport.authenticate("jwt", { session: false }),
+  (req, res) => {
+    if (req.user.role === "recruiter") {
+      getAllProfiles()
+        .then(data => {
+          res.send(data);
+        })
+        .catch(err => {
+          console.error(err);
+          res.send(500);
```

Comment:
This is probably not what you want as it doesn't send a response status of 500, it send `500` as the response. I suggest changing it to either `res.sendStatus(500)` or `res.status(500).send('some description here')`.

Reply:
res.send(Number) actually will res.sendStatus, whereas res.send(String), which is why we have some res.send(500) sitting around in the codebase and probably the syllabus. These should more cleanly be res.sendStatus(500) :)